Makefile.frankfurt

```diff
@@ -156,9 +156,15 @@ define build_templates
 	envsubst < 25-mf-chsdi3.conf.in > 25-mf-chsdi3.conf
 endef
 
+define setup_env
+	@echo "Setup vars from ENV_FILE=$(ENV_FILE)"
+	$(eval include $(1))
+	$(eval export sed 's/=.*//' $(1))
+endef
+
 local.ini: local.ini.in base.ini guard-OPENTRANS_API_KEY guard-PGUSER guard-PGPASSWORD guard-GIT_HASH_SHORT guard-APACHE_PORT
+	$(call setup_env,$(ENV_FILE))
```

Comment:
Would a guard-ENV_FILE make sense in that case?

Reply:
Yes.